cognite/client/__init__.py

```diff
@@ -1,3 +1,3 @@
 from cognite.client._cognite_client import CogniteClient
 
-__version__ = "1.8.1"
+__version__ = "1.9.0"
```

Comment:
Not sure about this. Based on [metrics from the last two weeks](https://docs.google.com/spreadsheets/d/1O_w-4T_csqp20on8UxVg1o687LQRbtATYFyBVygtrf4/edit?usp=sharing) we have quite a lot of usage on Labels. Unfortunately, we don't know if the Asset usage is using Labels.

I think we should bump major. Ultimately, this is a breaking change and we need users to trust that we won't break things on minors.

Reply:
I think a new major will make people hesitant to upgrade, and may lead to confusion that there is an api v2.

Reply:
> I think a new major will make people hesitant to upgrade, and may lead to confusion that there is an api v2.

We want to separate the SDK version from the API version (especially since the API will move into data versions while the SDK will remain on semantic versioning). For example, we're releasing JS SDK v3 this month.

Reply:
I think that break is better done when you actually move to date versions, rather than now.

Reply:
I'm a firm believer that if we break a running app/service with a minor release then it'll be hard to ask the developers to trust us in the future. This is a breaking change and we should provide that visibility along with an accurate description of what was changes so people feel safe upgrading.

Reply:
I am fine with both. Just a bid sad that we "waste" a major bump on such a silly mistake

Reply:
I bumped the version to 2.0.0 and updated the changelog